protected/humhub/modules/user/models/Password.php

```diff
@@ -80,7 +82,29 @@ public function rules()
             [['newPassword'], 'unequalsCurrentPassword', 'on' => 'changePassword'],
             [['newPasswordConfirm'], 'compare', 'compareAttribute' => 'newPassword', 'on' => 'changePassword'],
             [['newPasswordConfirm'], 'compare', 'compareAttribute' => 'newPassword', 'on' => 'registration'],
+			[['newPassword', 'newPasswordConfirm'], 'string', 'min' => Yii::$app->getModule('user')->settings->get('auth.MinPasswordLength'), 'max' => 255],
+			[['newPassword', 'newPasswordConfirm'], 'string', 'min' => 5, 'max' => 255],
+			[['newPassword'], PasswordSecurityValidator::className(), 'up' => 1, 'low' => 1, 'digit' => 1, 'spec' => 1],
```

Comment:
Instead of duplicate the code block, you can also use some Yii Validation Rule option:
http://www.yiiframework.com/doc-2.0/yii-validators-validator.html#$when-detail